Habinezajanvier/Capstone
# 2-about-me-page add resume page

#### What does this PR do?
Add resume on my website
#### Description of Task to be completed?
- Add markups of resume using HTML
- Style resume using CSS
#### How should this be manually tested?
- Clone the repo `https://github.com/Habinezajanvier/my-brand-project.git`
- Switch the directory `cd my-brand-project`
- Checkout the branch `git checkout ft-add-resume-page-2-about-me-page`
- In `/UI/` folder Start `index.html` file using your preferred browser
- In your opened browser (*janvierHabineze website*) Click on **Resume** found on navbar (*Top right part of the page*)
> You should see opened resume page of mine
#### Any background context you want to provide?
Opening `UI/index.html` file using liveServer should be helpful
#### What are the relevant pivotal tracker stories?
Click [here](https://trello.com/c/cCSXuspu/2-about-me-page) for **Trello** story description
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/44426710/88118886-1df4dc80-cbbf-11ea-9f7b-c119415e759d.png)
